### PR TITLE
Export parseQuery (#76)

### PR DESCRIPTION
**Pull request by anseki**
- Add `exports.parseQuery` in `lib/index.js`
- Separate `test/parseQuery.test.js` from `test/getOptions.test.js`
- Add `parseQuery` in `README.md`

### CODE REVIEW

File: test/getOptions.test.js [hunk 1]
```diff
@@ -4,72 +4,14 @@ const assert = require("assert");
...
-		}].forEach(test => {
-			it(test.it, () => {
-				assert.deepEqual(
-					loaderUtils.getOptions({
-						query: test.query
-					}),
-					test.expected
-				);
-			});
+	describe("when loaderContext.query is a string that has length", () => {
```
Comment (jhnns) [inline test/getOptions.test.js]: I'd write `... is a string with length > 0`

File: README.md [hunk 2]
```diff
@@ -36,10 +36,23 @@ someLibrary(options);
...
 
-If the loader options have been passed as loader query string (`loader?some&params`), the string is parsed like this:
+If the loader options have been passed as loader query string (`loader?some&params`), the string is parsed by using [`parseQuery`](#parsequery).
+
+### `parseQuery`
+
+Parses a passed string (e.g. `loaderContext.resourceQuery`) as a query string, and returns an object.
+
+``` javascript
+var params = loaderUtils.parseQuery(this.resourceQuery); // resource: `file?param1=foo`
```
Comment (jhnns) [inline README.md]: Please use `const` in the README if the value is not changed

File: README.md [hunk 3]
```diff
@@ -36,10 +36,23 @@ someLibrary(options);
...
+if (params.param1 === 'foo') {
```
Comment (jhnns) [inline README.md]: Please use double quotes

**Comment by anseki** [comment]
I did not change other `var`s in other parts, e.g. `urlToRequest`.

**Comment by jhnns** [comment]
Oh, that would be great. Didn't realize we already have this inconsistency 😁

**Comment by anseki** [comment]
Ok, I replaced those with `const`.

**Comment by jhnns** [comment]
Published as `1.1.0`

**Comment by anseki** [comment]
Thank you 😄

**Pull request merged**